Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,12 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            var demo = EventManager.ManageEvents();
+            demo.Run(); 
+                
```

Comment:
Lets keep code tidy :D no random blank lines!

Remove the blank lines from line 9, 12, 13, 14

Reply:
done